### PR TITLE
fix duplicate flux and remove todos

### LINKED ISSUE #57: fluxctl permissions are listed multiple times in the setup instructions
chmod a+x /usr/local/bin/fluxctl
    chmod a+x /usr/local/bin/fluxctl
    chmod a+x /usr/local/bin/fluxctl

Remove all but one

Also: please do some girlscout cleanup and remove TODO comments from the readme

### PR DESCRIPTION
**Pull request by TessFerrandez**
Closes #57 

- Remove duplicate fluxctl folder permissions
- Remove TODOs
- Change lists to 1 (numbering) to avoid future numbering problems

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -17,8 +17,6 @@ You need to install:
     sudo curl -L https://github.com/fluxcd/flux/releases/download/1.21.1/fluxctl_linux_amd64 -o
     /usr/local/bin/fluxctl
     chmod a+x /usr/local/bin/fluxctl
-    chmod a+x /usr/local/bin/fluxctl
-    chmod a+x /usr/local/bin/fluxctl
     ```
 
     > Note the version number in the URL above.
```
Comment (tompaana) [inline setup.md:22]:
```suggestion
```

Comment (tompaana) [reply setup.md:22]: and remove the extra empty line below too

**Review by Lybecker — APPROVED** [review]
Looks good :)

**Review by tompaana — APPROVED** [review]
Approving w/ suggestions

**Pull request merged**